.github/workflows/website-configs.yml

```diff
@@ -49,8 +49,17 @@ jobs:
       if: "steps.composer-cache.outputs.cache-hit != 'true'"
       run: "composer install --prefer-dist --no-progress --no-suggest --no-dev"
 
-    - name: "Prepare Webite files"
-      run: "bin/console --env=test sync-repositories && bin/console --env=test sync && bin/console --env=test clear-build-cache  && bin/console --env=test build-website-data"
+    - name: "Checkout repositories"
+      run: "bin/console --env=test sync-repositories"
+
+    - name: "Sync repositories"
+      run: "bin/console --env=test sync"
+
+    - name: "Clearing build cache"
+      run: "bin/console --env=test clear-build-cache"
```

Comment:
@SenseException likewise, is this really necessary? We're in the CI, so we are starting afresh, right?